qiskit_aqua/algorithms/components/multiclass/one_against_rest.py

```diff
@@ -20,16 +20,42 @@
 from sklearn.utils.validation import _num_samples
 from sklearn.preprocessing import LabelBinarizer
 
+from qiskit_aqua.algorithms.components.multiclass.multiclass_extension import MulticlassExtension
+
 logger = logging.getLogger(__name__)
 
-class OneAgainstRest:
+class OneAgainstRest(MulticlassExtension):
```

Comment:
the same problem, two lines between class declaration and others

Reply:
done